src/components/Card.module.css

```diff
@@ -50,3 +49,20 @@
   display: flex;
   align-self: end;
 }
+
+.flipVerticalRight {
+	animation: flip-vertical-right 0.4s cubic-bezier(0.455, 0.030, 0.515, 0.955) both;
+}
+
+@keyframes flip-vertical-right {
+  0% {
+    transform: rotateY(0);
+  }
+  50% {
+    transform: rotateY(90deg);
+  }
+  100% {
+    transform: rotateY(0);
+  }
```

Comment:
I'm really fond of this "hack". Initially I was rotating by 180 degrees, which caused everything to be a mirror-image, and the text was changing too soon. So, I added a timeout that makes the text change happen exactly in the middle - when the card is just a straight line - and made the animation only turn by 90 degrees, just to make it go back. Visually it tricks the mind that a flip is happening, but in reality it was really simple to implement.

This task reminded me of the mouse cursor animation during loading in the old game Battlefield 2. I cannot find a video link, but it was basically the same trick as the one I used here.